### ISSUE TITLE
issues upgrading from 1.24 to 1.25

### ISSUE THREAD

**Issue by NamantH** [Cacti/cacti#2900]
Unsure how to title this. Posted on the forums first.

I get stuck on the last step of the install/upgrade process.  I have tried with IE and chrome. I have cleared cache both times.  

Upgrade was initiated thru YUM update 

Upon getting to the Step 6 stage it will keep going back to that page on clicking next.

setting log_install to 5 reveled this

> [2019-08-15 08:49:46] [    snmp_options notice  ] snmp_option set:'3'
> [2019-08-15 08:49:46] [    snmp_options debug   ] Set snmp_option: install_snmp_option_snmp_retries = 3
> [2019-08-15 08:49:46] [      automation info    ] setProfile(1) returns with 1
> [2019-08-15 08:49:46] [      automation info    ] setCronInterval(300) returns with 300
> [2019-08-15 08:49:46] [          global notice  ] setTrueFalse(automation_mode, 'false' sets , returns 1
> [2019-08-15 08:49:46] [      automation info    ] setAutomationMode(false) returns with
> [2019-08-15 08:49:46] [      automation info    ] setAutomationRange() returns with
> [2019-08-15 08:49:46] [          global notice  ] setTrueFalse(automation_override, 'false' sets , returns 1
> [2019-08-15 08:49:46] [      automation info    ] setAutomationOverride(false) returns with
> [2019-08-15 08:49:46] [            step debug   ] Error: 6
> [2019-08-15 08:49:46] [            step debug   ] Done: 6
> [2019-08-15 08:49:46] [            json notice  ]   End: {"Mode":3,"Step":6,"Errors":{"Automation":{"Range":"Failed to apply specified Automation Range"}},"Eula":true,"Prev":{"Text":"Previous","Step":5,"
> Enabled":true,"Visible":true},"Next":{"Text":"Next","Step":7,"Enabled":true,"Visible":true},"Test":{"Text":"Test Connection","Step":-4,"Enabled":true,"Visible":false},"Theme":"modern","StepData":null}

I have tried reooting the machine. Restarting servers (httpd and mysqld)

First time thru (going in fresh after a cache write) I see the following in the url:
http://.../cacti/install/install.php?data={"Step":"6","Eula":"1"}

If i click next once i get:
http://.../cacti/install/install.php?data={"Step":6,"Eula":true}

if i click previous then next:
http://.../cacti/install/install.php?data={"Step":"6","Eula":true}

also on the paths screen I see a warning but the paths are all checked correct.
![stuck](https://user-images.githubusercontent.com/22033654/63110510-a1db2900-bf59-11e9-840f-1346f0340fb0.PNG)

Please let me know any other information you need to process the issue. I would rather not wipe and reinstall.

**Comment by netniV** [comment]
This appears to be more an issue with applying the automation values.  Do you have anything set for the automation network?

**Comment by NamantH** [comment]
I tried to disable that scan automation. I tried putting in a xxx.xxx.0.0. I ended up having to put in xxx.xxx.1.1 to get it to process to move forward. There was no error coming up on screen and I did try to toggle the scan off and that wouldnt work as well.

so It is installed and working using my "gateway" address as the automation... The 2 jobs for automation I installed previous to the upgrade are still in tact and working. Cannot see any trace of that numer I entered in the install/upgrade process
